src/morphology/english/getVerbForms.js

```diff
@@ -144,7 +144,11 @@ const checkIrregulars = function( word ) {
  * @returns {boolean} True if the word ends with "s".
  */
 const endsWithS = function( word ) {
-	return word[ word.length - 1 ] === "s";
+	const wordLength = word.length;
+	if ( wordLength > 3 ) {
```

Comment:
I think it would be good to add a short comment about the reason for the 3 letter boundary.